src/src/DataStructs/SettingsStruct.h

```diff
@@ -400,7 +412,24 @@ class SettingsStruct_tmpl
   uint8_t       Notification[NOTIFICATION_MAX] = {0}; //notifications, point to a NPLUGIN id
   // FIXME TD-er: Must change to pluginID_t, but then also another check must be added since changing the pluginID_t will also render settings incompatible
   uint8_t       TaskDeviceNumber[N_TASKS] = {0}; // The "plugin number" set at as task (e.g. 4 for P004_dallas)
-  unsigned int  OLD_TaskDeviceID[N_TASKS] = {0};  //UNUSED: this can be reused
+  int8_t        Pin_i2c2_sda = -1; // DEFAULT_PIN_I2C2_SDA; // From here, storage borrowed from OLD_TaskDeviceID array
+  int8_t        Pin_i2c2_scl = -1; // DEFAULT_PIN_I2C2_SCL;
+  int8_t        Pin_i2c3_sda = -1; // DEFAULT_PIN_I2C3_SDA;
+  int8_t        Pin_i2c3_scl = -1; // DEFAULT_PIN_I2C3_SCL;
+  uint32_t      I2C2_clockSpeed = 400000;
+  uint32_t      I2C2_clockSpeed_Slow = 100000;
+  uint32_t      I2C3_clockSpeed = 400000;
+  uint32_t      I2C3_clockSpeed_Slow = 100000;
+  int8_t        I2C2_Multiplexer_Type = I2C_MULTIPLEXER_NONE;
+  int8_t        I2C2_Multiplexer_Addr = -1;
+  int8_t        I2C2_Multiplexer_ResetPin = -1;
+  int8_t        I2C3_Multiplexer_Type = I2C_MULTIPLEXER_NONE;
+  int8_t        I2C3_Multiplexer_Addr = -1;
+  int8_t        I2C3_Multiplexer_ResetPin = -1;
+  uint16_t      I2C_peripheral_bus = 0;

```

Comment:
These newly added settings should be at the end of the settings struct, not somewhere inbetween and thus shifting other settings.
This will render existing settings unusable.

Reply:
see above reply...